package.json

```diff
@@ -5,7 +5,7 @@
   "publisher": "trufflesuite-csi",
   "preview": true,
   "icon": "images/TruffleLogo.png",
-  "version": "2.3.1",
+  "version": "2.3.5",
```

Comment:
@michaeljohnbennett after bumping to `2.4.0`, shouldn't we change the version here as well?